lib/x11regressiontest.pm

```diff
@@ -481,9 +481,20 @@ sub exit_firefox() {
 }
 
 sub start_gnome_settings {
-    send_key "super";
-    wait_still_screen;
-    type_string "settings";
+    my $is_sle_sp1 = check_var('VERSION', '12-SP1');
+    my $workaround_repetitions = 3;
+
+    do {
+        if ($is_sle_sp1) {
+            send_key "super";    # if launcher open, close (search string will also be removed).
```

Comment:
I don't understand the comment 'if launcher open, close'. Do you mean 'This will close an already open launcher'?